docs/application/native/guides/app-management/app-controls.md

```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
 
 The following common application controls are available:
 
-- [Browser](common-appcontrol.md#browser)
-- [Calendar](common-appcontrol.md#calendar)
-- [Call](common-appcontrol.md#call)
-- [Camera](common-appcontrol.md#camera)
-- [Contact](common-appcontrol.md#contact)
-- [Email](common-appcontrol.md#email)
-- [File Storage](common-appcontrol.md#file)
-- [Input Delegator](common-appcontrol.md#inputdelegator)
-- [Map](common-appcontrol.md#map)
-- [Message](common-appcontrol.md#message)
-- [Multimedia](common-appcontrol.md#multimedia)
-- [System Settings](common-appcontrol.md#settings_main)
-	- [Settings for Bluetooth](common-appcontrol.md#settings_bluetooth)
-	- [Settings for Location](common-appcontrol.md#settings_location)
-	- [Settings for NFC](common-appcontrol.md#settings_nfc)
-	- [Settings for Wi-Fi](common-appcontrol.md#settings_wifi)
-- [Voice Recorder](common-appcontrol.md#voice)
-- [VPN Service](common-appcontrol.md#vpnservice)
+- [Browser](common-appcontrols.md#browser)
+- [Calendar](common-appcontrols.md#calendar)
+- [Call](common-appcontrols.md#call)
+- [Camera](common-appcontrols.md#camera)
+- [Contact](common-appcontrols.md#contact)
+- [Email](common-appcontrols.md#email)
+- [File Storage](common-appcontrols.md#file)
+- [Input Delegator](common-appcontrols.md#inputdelegator)
+- [Map](common-appcontrols.md#map)
+- [Message](common-appcontrols.md#message)
+- [Multimedia](common-appcontrols.md#multimedia)
+- [System Settings](common-appcontrols.md#settings_main)
```

Comment:
It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_main` is old tag and removed now. To jump to the right location, it should be `system-settings`.